### PR TITLE
feat(discharge): redesigned discharge + chores

### LINKED ISSUE #100: bug(hvs): incorrect silkscreen for PS1
### Description
ps1 part has incorrect orientation of silk screen, but polarity is correct

### PR DESCRIPTION
**Pull request by Mxrxk**
### Changes
- Separated Hv from Lv
- Separated Hv+ and Hv- into 2 separate connectors
- Fixed Isolator footprint
- Renamed "discharge-breakout-pcb" to "Discharge"

### Related Issue(s)
Closes #100 

### Checklist
- [x] ERC passes with no errors.
- [x] DRC passes with no errors.
- [x] No files which should not be under version control added (see [README](https://github.com/sufst/pcb/blob/main/README.md)).
- [x] Library paths are relative paths.

### Additional Notes
There seems to be an issue in the DRC about the pads for the relay. Not sure what this is about.
There are also warnings about the Footprints not matching libraries for footprints with 3D models. Resolving them causes 3D models to disappear.

### CODE REVIEW

File: Discharge/Discharge.kicad_pcb
Comment (t-bre) [inline Discharge/Discharge.kicad_pcb:1]: The silkscreen needs a clear marking of what side is TS and what side is LV, e.g. with the HV sense: 

Screenshot 2023-04-29 at 18 52 23

Comment (t-bre) [reply Discharge/Discharge.kicad_pcb:1]: Silkscreen here looks misaligned

Screenshot 2023-04-29 at 18 53 10

Comment (t-bre) [reply Discharge/Discharge.kicad_pcb:1]: The GND trace spends a lot of time within that isolation area which I think might not be looked upon favourably at scrut. I'd be inclined to route it through the LV area first and then just have a small part of trace into the isolation area where absolutely necessary. 

Screenshot 2023-04-29 at 18 54 11

Comment (t-bre) [reply Discharge/Discharge.kicad_pcb:1]: The first DRC error indicates that the component was created as SMD type in the SUFST library, when it should be through hole. 

Screenshot 2023-04-29 at 18 56 15

You can fix this in the footprint properties:

Screenshot 2023-04-29 at 18 57 34

The rest of the DRC errors are a thing that's new to KiCad 7 - if you add a 3D model to a footprint, it considers the footprint as having changed from the library. I guessed either the default severity for this has been changed from "ignore" to "warning" or this is just a bug. Can be fixed by changing the severity to "ignore" in the miscellaneous design rules (in the board setup window):

Screenshot 2023-04-29 at 18 59 23

**Review by t-bre — CHANGES_REQUESTED** [review]
Small bits to change but otherwise looks good

**Comment by Mxrxk** [comment]
Changes:
- Labelled sides as "TS" and "LV"
- Fixed silkscreen miss alignment
- Shortened the amount of GND trace within isolation area
- Fixed relay footprint issues
- Changed the DRC check "Footprint doesn't match copy in library" to "ignore"
- Changed version from v1.0.2 to v1.1.0
- Edited sheet layout information

**Review by t-bre — COMMENTED** [review]
Looks good.

Made a small tweak to the silkscreen to replace "HV+" and "HV-" with "TS+" and "TS-", but committed that fix myself.

**Review by t-bre — APPROVED** [review]

**Pull request merged**